starwars/src/App.js

```diff
@@ -33,6 +34,7 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <StarWarsData Makers = {this.state.starwarsChars} />
```

Comment:
you are passing this.state.starwarsChars as "Makers" to the StarWarsData, looking at your file structure, you should be passing Makers to StarWarsList, I would also consider renaming "Makers" to maybe "starwarsChars" or "Chars" for easier readability. Then in the StarWarsList you can map each individual char to StarWarsData